src/main/config/conversion/import/iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl

```diff
@@ -102,6 +102,41 @@
     </xsl:copy>
   </xsl:template>
 
+  <!--Add default unclassified as security constraint if missing from metadata xml-->
+  <xsl:template match="gmd:MD_DataIdentification">
+    <xsl:copy copy-namespaces="no">
+      <xsl:choose>
+        <xsl:when test="gmd:resourceConstraints/gmd:MD_SecurityConstraints/gmd:classification/gmd:MD_ClassificationCode">
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:element name="gmd:resourceConstraints">
```

Comment:
I believe it would be cleaner to simply add the element instead of using xsl:element/xsl:attribute/xsl:text.
i.e. 
https://github.com/metadata101/iso19139.ca.HNAP/blob/16537ee38e8492d1fb5d971f150e0016357cb46e/src/main/plugin/iso19139.ca.HNAP/update-fixed-info.xsl#L239-L250